stubs/inertia/resources/js/app.js

```diff
@@ -2,12 +2,12 @@ require('./bootstrap');
 
 import Vue from 'vue';
 
-import { InertiaApp } from '@inertiajs/inertia-vue';
+import { app as InertiaApp, plugin as InertiaPlugin } from '@inertiajs/inertia-vue';
```

Comment:
What is `app` used for here? I see later we declare a `const app` as a different value?

Reply:
We renamed the named exports to be more inline with what's done in the JS community (removing the "Inertia" prefixes, and using title case for components). Technically `InertiaApp` still exists, but we've added `App` and `plugin` as well. For forwards compatibility, this change is good, and shouldn't interfere with `app` below, since it's just an alias.

That said, @claudiodekker, I'd recommend using the uppercase version of `App`.

```suggestion
import { App as InertiaApp, plugin as InertiaPlugin } from '@inertiajs/inertia-vue';
```

Reply:
Yeah, so, right here it's just used as the 'selected' thing to import, and it's directly being aliased to `InertiaApp`, meaning that as a variable it doesn't exist yet.

Basically, Jonathan decided a while back that he wanted to deprecate `Vue.use(InertiaApp)` in favor of importing the plugin directly, and at the same decided that it might be better to export more 'generics', as other libraries also do this.

Because this makes the `InertiaApp` itself semi-deprecated (not yet, but it could be at some point) and look inconsistent otherwise, I figured I might as well go ahead and alias this at the same time.